src/components/fields/select-histogram/types.ts

```diff
@@ -0,0 +1,14 @@
+import { SelectHistogramProps } from "@lifesg/react-design-system";
+import { TComponentOmitProps } from "../../frontend-engine";
+import { IHistogramValidationRule } from "../histogram-slider";
+import { IBaseFieldSchema } from "../types";
+
+type TCustomOptions = Pick<SelectHistogramProps, "rangeLabelPrefix" | "rangeLabelSuffix">;
+
+export interface ISelectHistogramSchema<V = undefined>
+	extends IBaseFieldSchema<"select-histogram", V, IHistogramValidationRule>,
```

Comment:
create `ISelectHistogramValidationRule` instead of relying on validation from histogram slider